libs/vm-modules/tests/unit/big_number_wrapper_tests.cpp

```diff
@@ -360,4 +348,28 @@ TEST_F(UInt256Tests, DISABLED_uint256_type_casts)
   EXPECT_TRUE(result_is_ok);
 }
 
+TEST_F(UInt256Tests, DISABLED_uint256_sha256_assignment)
```

Comment:
why is this test disabled?

Reply:
It will fail now, as the constructor-from-bytes of the UInt256   or   a serialization of SHA256().final() needs to be changed, but I believe such rework should be planned future.

Reply:
ok, could you add this justification as a comment to the test? and also create a ticket to deal with the constructor.

Reply:
There is a detailed comment inside the test code block :) Would it be better to move it outside?
Regarding the ticket - sure, I'll do it.